source/renderer/app/containers/status/InternetConnectionStatusDialog.js

```diff
@@ -23,8 +23,8 @@ export default class InternetConnectionStatusDialog extends Component<any> {
         ariaHideApp={false}
       >
         <InternetConnectionOfflineStatus
-          checking={checkingInternetConnection}
-          checkAgain={checkInternetConnectionStatus}
+          checking={isInternetConnectionChecking}
```

Comment:
@yakovkaravelov use `checkInternetConnectionRequest.isExecuting` and rename property to isCheckingInternetConnectionStatus

Reply:
@yakovkaravelov please use request flag instead computed value. Also don't forget to change property declaration name.